src/pipelines/rust_app.rs

```diff
@@ -125,6 +130,11 @@ impl RustApp {
             args.push("--bin");
             args.push(bin);
         }
+        if let Some(cargo_features) = &self.cargo_features {
+            args.push("--features");
+            args.push(cargo_features);
```

Comment:
Do you know if this will properly handle the case where a users specifies a space-separated list of features? I'm just concerned that if the subprocess call does not properly quote the items here, then cargo might not be able to properly interpret the intended behavior. Thoughts?

Reply:
That is a good question, I am not sure of the answer. I will do some testing this evening and let you know what I find.

Reply:
Copy that. I'm on a roll right now, so I'm going to merge, update one of the examples to support a few demo features, and then verify that it does indeed cover our use cases.

Thanks boss!